File: src/main/java/com/hagoapp/f2t/datafile/DataRow.java
/*
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */

package com.hagoapp.f2t.datafile;

import java.util.ArrayList;
import java.util.List;

public class DataRow {
    private int rowNo;

    private List<DataCell> cells = new ArrayList<>();

    public int getRowNo() {
        return rowNo;
    }

    public void setRowNo(int rowNo) {
        this.rowNo = rowNo;
    }

    public List<DataCell> getCells() {
        return cells;
    }
}

File: src/main/kotlin/com/hagoapp/f2t/DataRow.kt
/*
 * This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/.
 */

package com.hagoapp.f2t

/**
 * This class contains a row of a data table.
 *
 * @property rowNo  row index in whole data set
 * @property cells  data cells in this row
 * @author Chaojun Sun
 * @since 0.2
 */
data class DataRow(
    var rowNo: Long = 0,
    val cells: List<DataCell>
)
